src/EnergyPlus/Coils/CoilCoolingDXCurveFitOperatingMode.cc

```diff
@@ -178,11 +180,23 @@ void CoilCoolingDXCurveFitOperatingMode::size(EnergyPlusData &state)
     // Auto size condenser air flow to Total Capacity * 0.000114 m3/s/w (850 cfm/ton)
     DataSizing::DataConstantUsedForSizing = this->ratedGrossTotalCap;
     DataSizing::DataFractionUsedForSizing = 0.000114;
-    SizingString = "Rated Condenser Air Flow Rate";
+    SizingString = "Rated Condenser Air Flow Rate [m3/s]";
     TempSize = this->original_input_specs.rated_condenser_air_flow_rate;
     ReportSizingManager::RequestSizing(state, CompType, CompName, SizingMethod, SizingString, TempSize, PrintFlag, RoutineName);
     this->ratedCondAirFlowRate = TempSize;
 
+
+    if (this->condenserType != AIRCOOLED) {
+        // Auto size Nominal Evaporative Condenser Pump Power to Total Capacity * 0.004266 w/w (15 W/ton)
+        SizingMethod = DataHVACGlobals::AutoCalculateSizing;
+        DataSizing::DataConstantUsedForSizing = this->ratedGrossTotalCap;
+        DataSizing::DataFractionUsedForSizing = 0.004266;
+        SizingString = "Nominal Evaporative Condenser Pump Power [W]";
+        TempSize = this->original_input_specs.nominal_evap_condenser_pump_power;
+        ReportSizingManager::RequestSizing(state, CompType, CompName, SizingMethod, SizingString, TempSize, PrintFlag, RoutineName);
+        this->nominalEvaporativePumpPower = TempSize;
+    }
+
```

Comment:
Autosize per IO ref at 15 W/ton.

Reply:
OK, so this is just using autocalculate, makes sense.  Per @rraustad comment, once they are merged together, this code change will no longer compile.  It will just require altering this code to use an AutocalculateSizer class instance instead.